.github/docker-performance-tests/alarms-poller/poll-during-performance-tests.py

```diff
@@ -0,0 +1,288 @@
+import argparse
+import logging
+import sys
+import time
+
+import boto3
+import docker
+
+logging.basicConfig(level=logging.INFO)
+
+logger = logging.getLogger(__file__)
+
+# AWS Client API Constants
+
+PROCESS_COMMAND_LINE_DIMENSION_NAME = "process.command_line"
+METRIC_DATA_STATISTIC = "Sum"
+
+COMMON_ALARM_API_PARAMETERS = {
+    "EvaluationPeriods": 5,
+    "DatapointsToAlarm": 3,
+    "ComparisonOperator": "GreaterThanOrEqualToThreshold",
+    "TreatMissingData": "ignore",
+}
+
+CPU_LOAD_ALARM_NAME = "OTel Performance Test - CPU Load Percentage Spike - Python"
+TOTAL_MEMORY_ALARM_NAME = (
+    "OTel Performance Test - Virtual Memory Usage Spike - Python"
+)
+
+# Docker Client API Constants
+
+LOAD_GENERATOR_CONTAINER_NAME = "docker-performance-tests_load-generator_1"
+APP_CONTAINER_NAME = "docker-performance-tests_app_1"
+COLLECTOR_CONTAINER_NAME = "docker-performance-tests_otel_1"
```

Comment:
qq: why these names end with "_1"?

Reply:
Not too sure why they need the "_1", but I found that this is what works to figure out if those containers are running both locally and on GitHub workflows. Anything else will produce a `KeyError`.